src/Field/Base/InputData/InputDataInterface.php

```diff
@@ -5,48 +5,67 @@
 namespace Yiisoft\Form\Field\Base\InputData;
 
 /**
- * Represents input data.
+ * Represents some common information about input and its validation rules / state.
  */
 interface InputDataInterface
 {
     /**
-     * @return string|null Name of the input or null if ???
+     * Unique name of an input within the form (not visible to the end user).
+     *
+     * @return string|null Name of the input or null if missing.
```

Comment:
When does it make sense for field name to be absent?

Reply:
One of the cases is using Javascript to handle data. Plus HTML allows this.